backpack.tf extended sorting.user.js

```diff
@@ -1268,4 +1268,246 @@ class</a></li>
+                "Crusader's Crossbow" : 305,
+                "Medi Gun" : 211,
+                "Bonesaw" : 198,
+                "Ubersaw" : 37,
+                "Sniper Rifle" : 201,
+                "Huntsman" : 56,
+                "SMG" : 203,
+                "Revolver" : 210,
+                "Ambassador" : 61,
+                "Knife" : 194,
+            }
+        }
+
+        getItemName(){
+            let itemName = this.name;
+            if(this.aussie == 1){
+                itemName = "Australium".concat(" ", itemName);
+            }
+            return itemName;
+        }
+
+        getItemId(){
+            let itemId;
+            let itemName = this.name.match(/((?<=((Blood)|(Gold)|(Silver)|(Rust)|(Carbonado)|(Diamond)) Botkiller ).+(?= Mk.II?))|((?<=Festive ).+)/g); // Regex courtesy of Moder112
+            if(itemName){
+                this.defindex = this.nameiddict[itemName[0]];
+                itemId = this.defindex;
+            }
+            else{
+                let defindexInt = parseInt(this.defindex);
+                if(this.idreplacedict.hasOwnProperty(defindexInt)){
+                    itemId = this.idreplacedict[defindexInt].toString();
+                }
+                else{
+                    itemId = this.defindex;
+                }
+            }
+            return itemId;
+        }
+
+        getItemSlot(){
+            let itemSlot;
+            switch(this.slot){
+                case "primary": case "secondary": case "melee": case "pda":
+                    itemSlot = 1;
+                    break;
+                case "misc":
+                    itemSlot = 0;
+                    break;
+                case "none":
+                    if(this.name == "Kit"){
+                        itemSlot = -1;
+                    }
+                    else if(this.name == "Fabricator"){
+                        itemSlot = -2;
+                    }
+                    else{
+                        itemSlot = 0;
+                    }
+                    break;
+            }
+            return itemSlot;
+        }
+
+        getKillstreakTier(){
+            return this.killstreak_tier;
+        }
+    };
+
+    if (window.location.pathname.startsWith("/stats")) {
+        const currentItem = new itemData(document.querySelector(".item"));
+        let itemId = currentItem.getItemId();
+        let killstreakTier = currentItem.getKillstreakTier();
+        let currentPath = window.location.pathname;
+
+        $(".stats-header-controls").append(`
+     <br>
+     <div class="btn-group btn-group-sm stats-killstreak-list" style="margin: 0 0 12px"></div>
+     `);
+
+        switch(currentItem.getItemSlot()){
+
+            case 0:
+                break;
+
+            case -1: {
+
+                let weaponName = trimSpaces(document.querySelector(".stats-breadcrumbs").children[1].innerHTML);
+                let weaponDefIndex = window.location.pathname.match(/\d+$/g);
```

Comment:
This method works perfectly fine, but it would additionally possible to use:

```suggestion
                let weaponDefIndex = $(".stats-header-item .item").attr("data-priceindex").split("-").pop()
```
-------------------------------
Exactly the same code (`$(".stats-header-item .item").attr("data-priceindex").split("-").pop()`) would work for fabricators too.